modules/og_context/og_context.info.yml

```diff
@@ -0,0 +1,7 @@
+name: OG context
+type: module
+description: Get a group from a viewed page.
+core: 8.x
+package: OG
+dependencies:
+  - og_ui
```

Comment:
We thought about it a bit this morning and we might don't need this as an external module. OG context is on of the strogest module in the OG stack. Why not put in inside the core?